README.md

```diff
@@ -10,10 +10,46 @@ Official Archethic Javascript library for Node and Browser.
 npm install archethic
 ```
 
+
 ## Usage
 
+When it comes to private data manipulation, your application has two options : 
+  1. **Standalone :** store private keys on your own
+  2. **WalletRPC :** delegate sensitive operations to a Wallet application. (**recommended**, but still Alpha)
+
+The second option is strongly preferable : it preserves you from security problematics. When your web application needs to perform a sensitive operation (signing a Transaction, reading a Transaction secrets), it will asks **Archethic Wallet** application to perform it.
+That way, cryptographic secrets never leaks from the **Archethic Wallet**.
+
+The only requirement is user has to run an **Archethic Wallet** on its computer.
+
+> If your application does not perform any sensitive operation, **WalletRPC** is probably not necessary.
+
+### Using the network with WalletRPC
+
 This library aims to provide a easy way to interact with Archethic network.
 
+
+```js
+import Archethic from "archethic";
+
+const archethic = new Archethic("ws://localhost:12345") // Endpoint is the Archethic Wallet RPC server
```

Comment:
What happens for the other queries such as `getTransactionIndex`, `getOracleData` ?
I didn't see any forward or leverage of the `getEndpoint()` 
Does it mean it will require two archethic instances, one with the json rpc connection and one with the direct endpoint ?

Reply:
`getEndpoint()` gets automatically resolved when calling `Archethic.connect()`.

That way, that `libjs` evolution is fully retrocompatible.

Reply:
Ok